### DISCUSSION TITLE
1.18.2

### DISCUSSION THREAD

**Discussion by KeiMuriKoe** [ldtteam/minecolonies#8919] [Q&A]
I was working on adapting MineColonies for TFC. I'm doing great thanks to your wiki page. Then I ran into one problem, I need builders to accept tools from the mod (for example, instead of a wooden shovel, a stone shovel or copper shovel from TFC). In your discord server told me that using datapack to do this will not work, how can I implement it?
![123](https://user-images.githubusercontent.com/106619492/215511275-28817e7a-ce44-470d-a5e9-7b7f73efc5c9.PNG)

**Comment by FokC352** [comment]
Hit me up on Discord FokC#2013
